extensions/adobe/experience/offer-management/rank.schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/offer-management/rank",
+  "type": "object",
+  "title": "Offer Rank",
+  "description": "Offer rank (priority)",
+  "definitions": {
+    "rank": {
+      "properties": {
+        "xdm:priority": {
+          "type": "integer",
+          "title": "Priority",
+          "description": "Offer priority",
+          "minimum": 0,
+          "maximum": 100
```

Comment:
Why 100? We use floats `0..1` everywhere else, and I think they are good enough for ranking.

Reply:
This is an end-user supplied value. I am not sure that asking users to enter a 0.xx is a reasonable workflow. We can convert, of course, but why?

Reply:
upper bound is removed now.